src/components/timeline/Markers.js

```diff
@@ -408,7 +408,6 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
           // browser window.
           canvas.setCapture();
         }
-        e.stopPropagation();
```

Comment:
I did some archeology, and this was present since markers in the timeline were implemented.